ruby/files_and_serialization/project_event_manager.md

```diff
@@ -1236,12 +1225,11 @@ The code above loads the ERB library, then creates a new ERB template with the
 results of the variable `meaning_of_life`. We send the `result` message to the
 template with `binding`.
 
-* What is `binding`?
+- What is `binding`?
 
-The method
-[binding](https://docs.ruby-lang.org/en/3.2/Kernel.html#method-i-binding)
-returns a special object. This object is an instance of
-[Binding](https://docs.ruby-lang.org/en/3.2/Binding.html). An instance of binding knows
+The [`Kernel#binding` method](https://docs.ruby-lang.org/en/3.2/Kernel.html#method-i-binding)
+returns a special object. This object is an instance of the
+[`Binding` class](https://docs.ruby-lang.org/en/3.2/Binding.html). An instance of binding knows
```

Comment:
These make sense! Just removing the backticks for the same rendering reason as above.
```suggestion
The [Kernel#binding method](https://docs.ruby-lang.org/en/3.2/Kernel.html#method-i-binding)
returns a special object. This object is an instance of the
[Binding class](https://docs.ruby-lang.org/en/3.2/Binding.html). An instance of binding knows
```